tests-web/www/js/tests.js

```diff
@@ -1,58 +1,100 @@
-import { makeOsmLayer, makeJsonLayer, makeDotLayer  } from './layers.js';
+import {
+  makeOsmLayer,
+  makePlainGeoJsonLayer,
+  makeDetailedGeoJsonLayer,
+  makeDotLayer,
+} from "./layers.js";
+import { JsStreetNetwork } from "./osm2streets-js/osm2streets_js.js";
+
+var currentLayers = [];
```

Comment:
I started a little bit of layer management here, but it doesn't work across everything. For example, we only create the layers widget after clicking on a test case.